packages/hash/frontend/src/blocks/page/BlockView.tsx

```diff
@@ -218,7 +237,7 @@ export class BlockView implements NodeView<Schema> {
               ),
             );
 
-            this.editorView.dispatch(tr);
+            // this.editorView.dispatch(tr);
```

Comment:
This is probably what has caused drag and drop to stop working
```suggestion
            this.editorView.dispatch(tr);
```